core/services/log/broadcaster.go

```diff
@@ -2,606 +2,105 @@ package log
 
 import (
 	"context"
-	"math/big"
-	"reflect"
 	"time"
 
-	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/pkg/errors"
-	"github.com/tevino/abool"
 
-	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/services/eth"
 	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
-//go:generate mockery --name Broadcaster --output ../../internal/mocks/ --case=underscore --structname LogBroadcaster --filename log_broadcaster.go
-//go:generate mockery --name Listener --output ../../internal/mocks/ --case=underscore --structname LogListener --filename log_listener.go
-//go:generate mockery --name Broadcast --output ../../internal/mocks/ --case=underscore --structname LogBroadcast --filename log_broadcast.go
+//go:generate mockery --name Broadcaster --output ./mocks/ --case=underscore --structname Broadcaster --filename broadcaster.go
```

Comment:
Personally, I think it's clearer in the tests to write `new(mocks.LogBroadcaster)` instead of `new(mocks.Broadcaster)` - there's no name collision now, but in the future maybe we have a "txReceiptBroadcaster" or something that will conflict. Not hellbent on it though.

Reply:
The way I've been dealing with this is to give each package its own `mocks` subpackage and then do something like the following:

```go
import (
    logmocks "github.com/smartcontractkit/chainlink/core/services/log/mocks
)

func TestBlah(t *testing.T) {
    lb := new(logmocks.Broadcaster)
}
```